pom.xml

```diff
@@ -148,86 +116,96 @@
-
     <dependency>
       <artifactId>java-session-handler</artifactId>
       <groupId>uk.gov.companieshouse</groupId>
       <version>${java-session-handler.version}</version>
     </dependency>
-    
-    
     <dependency>
       <groupId>uk.gov.companieshouse</groupId>
       <artifactId>web-security-java</artifactId>
       <version>${web-security-java.version}</version>
     </dependency>
-    
     <dependency>
       <artifactId>oauth-signin-java-library</artifactId>
       <groupId>uk.gov.companieshouse</groupId>
       <version>${oauth-signin-java.version}</version>
     </dependency>
-    
         <!-- Security dependencies -->
     <dependency>
       <artifactId>spring-security-core</artifactId>
       <groupId>org.springframework.security</groupId>
     </dependency>
-
     <dependency>
       <artifactId>spring-security-web</artifactId>
       <groupId>org.springframework.security</groupId>
     </dependency>
-
     <dependency>
       <artifactId>spring-security-config</artifactId>
       <groupId>org.springframework.security</groupId>
     </dependency>
   </dependencies>
-  <description>Static content for the developer hub,
-    such as "getting started" and "introduction" documentation
-  </description>
 
-  <modelVersion>4.0.0</modelVersion>
-  <name>docs.developer.ch.gov.uk</name>
-
-  <packaging>jar</packaging>
-
-  <parent>
-    <artifactId>companies-house-parent</artifactId>
-    <groupId>uk.gov.companieshouse</groupId>
-    <version>1.2.0</version>
-    <relativePath/>
-  </parent>
 
-  <properties>
-    <common-web-java.version>1.1.0-rc1</common-web-java.version>
-    <java-session-handler.version>2.2.0</java-session-handler.version>
-    <java.version>1.8</java.version>
-    <junit-jupiter-engine.version>5.5.2</junit-jupiter-engine.version>
-    <maven.compiler.source>${java.version}</maven.compiler.source>
-    <!--    These versions are earlier than in the Parent POM-->
-    <maven.compiler.target>${java.version}</maven.compiler.target>
-    <mockito-junit-jupiter.version>3.1.0</mockito-junit-jupiter.version>
-
-    <structured-logging.version>1.5.0-rc2</structured-logging.version>
-    <thymeleaf-extras-java8time.version>3.0.1.RELEASE</thymeleaf-extras-java8time.version>
-
-    <thymeleaf-layout-dialect.version>2.3.0</thymeleaf-layout-dialect.version>
-
-    <web-security-java.version>1.3.8</web-security-java.version>
-    <oauth-signin-java.version>0.2.2</oauth-signin-java.version>
-
-    <spring-boot-dependencies.version>2.2.0.RELEASE</spring-boot-dependencies.version>
-    <spring-test.version>5.1.3.RELEASE</spring-test.version>
-    <spring-boot-maven-plugin.version>2.2.0.RELEASE</spring-boot-maven-plugin.version>
-    <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
-    <jacoco-maven-plugin.version>0.8.4</jacoco-maven-plugin.version>
-
-    <home.url></home.url>
-    <junit-platform-surefire-provider.version>1.2.0</junit-platform-surefire-provider.version>
-  </properties>
 
+  <build>
```

Comment:
Moved and indentation changes for some of the top lines (Unless I did a copy paste error)